src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let operationsCount = 0;
+
+  const robotAccountant = (a) => {
```

Comment:
[Checklist] - You don't need to name functions if you return them right away

Reply:
I appreciate your advice, but I think this rule is more about the cases when we declare and return function at the same line, such as `return function robot() {};`.
However, in my particular case, I make sure to enhance code readability and comprehension by creating a function and giving it a clear and understandable name before returning it.